### PR TITLE
SkyBrowser Hash Handling

### PR DESCRIPTION
**Pull request by alexanderbock**
This PR does some minor cleanup of the SkyBrowser module but more importantly adds the ability to compare a local hash to a remote hash to perform automatic updating of files.  It also moves the wwtimagedata folder from the modules/skybrowser into the sync folder

### CODE REVIEW

File: modules/skybrowser/src/utility.cpp [hunk 1]
```diff
@@ -150,11 +167,10 @@ bool isCoordinateInView(const glm::dvec3& equatorial) {
...
     glm::dvec3 localCamera = equatorialToLocalCamera(equatorial);
     glm::dvec3 coordsScreen = localCameraToScreenSpace3d(localCamera);
-    double r = static_cast<float>(windowRatio());
-
-    bool isCoordInView = abs(coordsScreen.x) < r && abs(coordsScreen.y) < 1.f &&
-                         coordsScreen.z < 0;
+    double r = windowRatio();
 
+    bool isCoordInView =
+        abs(coordsScreen.x) < r && abs(coordsScreen.y) < 1.f && coordsScreen.z < 0;
```
Comment (WeirdRubberDuck) [inline modules/skybrowser/src/utility.cpp:173]: `coordsScreen.z < 0.f` ? Don't remember if we do an exception for 0

**Review by WeirdRubberDuck — APPROVED** [review]
Looks good to me and runs without problems

Great update!

**Review by ylvaselling — APPROVED** [review]
Tried it, works well!

File: modules/skybrowser/src/utility.cpp (hunk 1 above)
Comment (alexanderbock) [reply modules/skybrowser/src/utility.cpp:173]: No, we don't... you're right

**Pull request merged**